src/PackageCompiler.jl

```diff
@@ -1183,6 +1183,68 @@ function glob_pattern_lib(lib)
     error("unknown os")
 end
 
+# Take `path` to a libstdc++ library and return aliases for soname symlinks
+# - Unix:
+#   `path/to/libstdc++.so.6.0.30` will yield aliases
+#   `libstdc++.so.6.0.30`, `libstdc++.so.6`, `libstdc++.so`
+# - Apple:
+#   `path/to/libstdc++.6.0.30.dylib` will yield aliases
+#   `libstdc++.6.0.30.dylib`, `libstdc++.6.dylib`, `libstdc++.dylib`
+# - Windows (no versions in filenames):
+#   `path/to/libstdc++.dll` will yield alias `libstdc++.dll`
+function get_libstdcxx_aliases(path)
```

Comment:
If you like, you can make use of `Base.BinaryPlatforms.parse_dl_name_version` to eliminate some of the toil here.

Reply:
Thanks for the hint! Done in https://github.com/JuliaLang/PackageCompiler.jl/pull/853/commits/8fbe560a534d58a4a0ae3d8e191e383ab1eee1ab.